test2.js

```diff
@@ -0,0 +1,5 @@
+This
+is
+a
```

Comment:
Files Panel -> Clicked on a file -> Start review

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix